Team9Project/res/menu/add_tag_to_claim.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item android:id="@+id/action_add_tag_to_claim_string" android:alphabeticShortcut="@string/action_add_tag_to_claim" android:title="@string/action_add_tag_to_claim"></item>
```

Comment:
Should be reindented in Eclipse (splitting attributes on multiple lines) for readability